web/admin.py

```
@@ -812,3 +818,131 @@ class PointsAdmin(admin.ModelAdmin):
+    list_filter = ("status", "applied_at", "processed_at")
+    search_fields = ("user__username", "user__email", "chapter__name", "motivation")
+    raw_id_fields = ("user", "chapter", "processed_by")
+    readonly_fields = ("applied_at", "processed_at")
+    fieldsets = (
+        (None, {"fields": ("user", "chapter", "status")}),
+        ("Application Details", {"fields": ("motivation", "experience", "skills")}),
+        ("Processing", {"fields": ("processed_by", "processed_at", "rejection_reason")}),
+        ("Timestamps", {"fields": ("applied_at",), "classes": ("collapse",)}),
+    )
+    actions = ["approve_selected", "reject_selected"]
+    def approve_selected(self, request, queryset):
```

Ruff (0.8.2)

Missing return type annotation for public function `approve_selected`

Add return type annotation: `None`

(ANN201)